### PR TITLE
Fix deserialization error of deleted Object in EventLoggerJob

### PR DESCRIPTION
**Pull request by x4d3**
Delete object (for example widget) could not be found during deserialization in the EventLoggerJob and nil exception were happening.

### CODE REVIEW

**Comment by x4d3** [comment]
@ouranos Could you please review

**Comment by ouranos** [comment]
@x4d3 I still think there's a problem with the method calls:

5 args
```
MnoEnterprise::EventLogger.info('app_destroy', current_user.id, 'App destroyed', {name: app_instance.name, app_nid: app_instance.app.nid}, app_instance)
```

Expecting 6 args without default:
```
def info(key, current_user_id, description, subject_type, subject_id, metadata)
```

Could you also rebase from `master`?

**Comment by x4d3** [comment]
@ouranos MnoEnterprise.EventLogger.info is taking 5 arguments
`def self.info(key, current_user_id, description, metadata, object)`

EventLoggerJob, IntercomEventsListener and AuditEventsListener are taking 6.

```
def self.info(key, current_user_id, description, metadata, object)
  formatted_metadata = format_metadata(metadata, object)
  subject_type = object.class.name
  subject_id = object.id
  # TODO: improve
  # Bypass Job queuing in specs or we'd have to stub lots of Her call for the deserialization
  if Rails.env.test?
    self.send_info(key, current_user_id, description, subject_type, subject_id, formatted_metadata)
  else
    MnoEnterprise::EventLoggerJob.perform_later('info', key, current_user_id, description, subject_type, subject_id, formatted_metadata)
  end
rescue ActiveJob::SerializationError
  Rails.logger.warn "[MnoEnterprise::EventLogger] Serialization error, skipping #{key} event"
end
```

**Comment by ouranos** [comment]
Ok my bad. Looks good then.

Instead of passing a hash for the metadata, why not rely on the automatic call to `#to_audit_event` in `formatted_metadata`?

**Comment by x4d3** [comment]
@ouranos Could you please review again, I took your idea of using 'to_audit_event' and I improved the code.

File: core/app/models/mno_enterprise/user.rb [hunk 1]
```diff
@@ -261,4 +261,7 @@ def self.create_from_omniauth(auth, opts = {})
       user
     end
   end
+
+
+
```
Comment (ouranos) [inline core/app/models/mno_enterprise/user.rb]: Empty lines :scream_cat:

**Pull request merged**